src/app/util/af-main-common.cpp

```diff
@@ -474,8 +474,9 @@ EmberStatus emberAfSendUnicastToBindingsWithCallback(EmberApsFrame * apsFrame, u
         if (binding.type == EMBER_UNICAST_BINDING && binding.local == apsFrame->sourceEndpoint &&
             binding.clusterId == apsFrame->clusterId)
         {
-            apsFrame->destinationEndpoint = binding.remote;
-            status = send(EMBER_OUTGOING_VIA_BINDING, MessageSendDestination(i), apsFrame, messageLength, message,
+            apsFrame->destinationEndpoint            = binding.remote;
+            const MessageSendDestination destination = MessageSendDestination::ViaBinding(i);
```

Comment:
Fwiw, here and other places `auto` might have been nice... not a big deal either way.